pkgs/development/python-modules/libusb-package/default.nix

```
@@ -0,0 +1,74 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, libusb1
+, pytestCheckHook
+, python
+, pythonOlder
+, setuptools-scm
+, importlib-resources
+, pyusb
+}:
+
+buildPythonPackage rec {
+  pname = "libusb-package";
+  version = "1.0.26.2";
+  format = "setuptools";
+  disabled = pythonOlder "3.7";
+
+  src = fetchFromGitHub {
+    owner = "pyocd";
+    repo = "libusb-package";
+    rev = "v${version}";
+    hash = "sha256-bM+v3eeQQHiBKsFky51KdDYlRjFzykQFkuFKkluqkjY=";
+  };
+
+  patches = [ ./fix-pytest.patch ];
+
+  # Replace the setup to not build libusb
+  postPatch = ''
+    cp -f ${./setup.py} setup.py
+  '';
+
+  SETUPTOOLS_SCM_PRETEND_VERSION = version;
+  buildInputs = [
+    setuptools-scm
+  ];
+
+  propagatedBuildInputs = [
+    importlib-resources
+  ];
+
+  propagatedNativeBuildInputs = [
+    libusb1
+  ];
+
+  # Symlink the system libusb instead of building it
+  postInstall = ''
+    pushd "$out/${python.sitePackages}/libusb_package"
+    rm -f libusb-1.0.so
+    ln -s ${libusb1}/lib/libusb-1.0.so libusb-1.0.so
+    popd
```

The `pushd` and `popd` seems a little unnecessary when you could just work with the path directly

 ```suggestion
    rm -f "$out/${python.sitePackages}/libusb_package/libusb-1.0.so"
    ln -s "${libusb1}/lib/libusb-1.0.so" "$out/${python.sitePackages}/libusb_package/libusb-1.0.so"
```